docs-src/cloud/tcld/namespace/accepted-client-ca/add.md

```diff
@@ -28,7 +28,7 @@ Alias: `-n`
 tcld namespace accepted-client-ca add --namespace <namespace_id> --ca-certificate <encoded_certificate>
 ```
 
-#### `--request-id`
+#### --request-id
```

Comment:
The three following H4s (--resource-version, --ca-certificate, and --ca-certificate-file) still need their backticks extracted.